scipy/linalg/tests/test_special_matrices.py

```diff
@@ -640,3 +642,45 @@ def test_fiedler_companion():
     fc = fiedler_companion([1., -16., 86., -176., 105.])
     assert_array_almost_equal(eigvals(fc),
                               np.array([7., 5., 3., 1.]))
+
+
+def test_convolution_matrix():
+    'test convolution_matrix vs. numpy.convolve for various parameters'
+
+    def test_vector(n, cpx):
+        'make a complex or real test vector of length n'
+        if cpx:
+            return np.random.normal(size=(n, 2))\
```

Comment:
we try to avoid these slash continuations usually -- I guess that can be cleaned up later